stdlib/http/http.go

```diff
@@ -0,0 +1,235 @@
+package http
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/iocounter"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+)
+
+// maxResponseBody is the maximum response body we will read before just discarding
+// the rest. This allows sockets to be reused.
+const maxResponseBody = 512 * 1024 // 512 KB
+
+var httpGetStatusCodeOnly = values.NewFunction(
+	"get",
+	semantic.NewFunctionPolyType(semantic.FunctionPolySignature{
+		Parameters: map[string]semantic.PolyType{
+			"url":     semantic.String,
+			"headers": semantic.Tvar(1),
+			"data":    semantic.Bytes,
+		},
+		Required: []string{"url"},
+		Return:   semantic.Int,
```

Comment:
How do you plan to use the `http.get` function since it only returns the status code of the request but otherwise ignores the response?

Reply:
This get is almost a straight copy of the flux post request code that only returns statusCode.  My next plan is to implement get(url: "", headers: "", responseType: "string|json")